00_Utilities/TODO.md

```diff
@@ -0,0 +1,98 @@
+# TODO list
+ game | csharp | java | javascript | kotlin | lua | perl | python | ruby | rust | vbnet
+--- | --- | --- | --- | --- | --- | --- | --- | --- | --- | ---
+01_Acey_Ducey                  | ✅ | ✅ | ✅ | ⬜️ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅
+02_Amazing                     | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ✅ | ✅ | ✅
+03_Animal                      | ✅ | ✅ | ✅ | ✅ | ⬜️ | ✅ | ✅ | ✅ | rust | ✅
```

Comment:
Cool, but whats's up with the "rust" string in the table?

Reply:
It was a bug in the original script. I've just fixed it. It happened when the game-folder didn't contain the language folder.